autogen/agentchat/groupchat.py

```diff
@@ -131,3 +136,48 @@ def run_chat(
             speaker.send(reply, self, request_reply=False)
             message = self.last_message(speaker)
         return True, None
+
+    async def a_run_chat(
```

Comment:
add a test for this?

Reply:
Kniru's (my startup) product launch is on Oct 15th so will be occupied with it till then. I'll only be able to add after that but happy to include more thorough testing.

Reply:
Thanks @aayushchhabra1999! Will you be able to add a test for this soon?

Reply:
@Rysias, since you requested this feature in issue #103, could you help review and potentially also add a test (for the `a_run_chat` func) such that we can merge this PR sooner?

@LittleLittleCloud this async function we need to test is in group chat, would you be interested in testing this async func?

Your help would be greatly appreciated!

Reply:
I'd love to help! I have some time tomorrow evening (CET)

Reply:
@Rysias that would be great!

Reply:
@Rysias @gagb Thanks for the help and for taking over.
@qingyun-wu Apologies, on the road for another week.

Reply:
resolving and merging because tests have been made by @gagb @bonadio